third_party/move/move-compiler/src/inlining/translate.rs

```diff
@@ -725,9 +735,12 @@ impl<'l> Inliner<'l> {
             },
         };
 
+        let spanned_lvalues = sp(loc, lvalues);
+        let lvalue_ty = lvalues_expected_types(&spanned_lvalues); // BUGBUG - added
```

Comment:
What does this comment mean?

Reply:
Oops, was marking my place during debugging.